src/Traits/ReflectionClassLikeTrait.php

```diff
@@ -442,7 +443,7 @@ public function getParentClass()
             $extendsNode = $hasExtends ? $this->classLikeNode->$extendsField : null;
             if ($extendsNode instanceof FullyQualified) {
                 $extendsName = $extendsNode->toString();
-                $parentClass = class_exists($extendsName, false) ? new parent($extendsName) : new static($extendsName);
+                $parentClass = class_exists($extendsName, false) ? new BaseReflectionClass($extendsName) : new ReflectionClass($extendsName);
```

Comment:
Please revert, because Scrutinizer CI seem to incorrectly tracked trait usage in https://scrutinizer-ci.com/g/goaop/parser-reflection/inspections/eaf283f6-f982-44a4-bb28-51489bcba537/issues/files/src/Traits/ReflectionClassLikeTrait.php?status=fixed&orderField=path&order=asc&honorSelectedPaths=0.

The original code doesn't have any issues.